.github/workflows/build.yml

```
@@ -180,14 +180,17 @@ jobs:
         uses: actions/github-script@v6
         with:
           script: |
-            github.rest.repos.createCommitStatus({
+            github.rest.checks.create({
               owner: context.repo.owner,
               repo: context.repo.repo,
-              sha: '${{ github.event.pull_request.head.sha }}',
-              state: 'success',
-              context: 'Deploy preview',
-              description: 'Live preview of the current branch',
-              target_url: 'https://itwin.github.io/iTwinUI/${{ github.event.number }}',
+              head_sha: '${{ github.event.pull_request.head.sha }}',
+              conclusion: 'neutral',
+              name: 'deploy-preview',
+              output: {
+                title: 'Deploy preview',
+                summary: 'Live preview of the current branch',
```

This would look better maybe?
```suggestion
              name: 'Deploy preview',
              output: {
                title: 'Live preview of the current branch',
```